src/pages/workspaces/List.js

```diff
@@ -191,16 +182,15 @@ export const WorkspaceList = ajaxCaller(class WorkspaceList extends Component {
           ]),
           viewToggleButtons(listView, listView => this.setState({ listView }))
         ]),
-        div({ style: styles.cardContainer }, [
+        div({ style: styles.cardContainer(listView) }, [
           h(NewWorkspaceCard, {
-            listView,
             onClick: () => this.setState({ creatingNewWorkspace: true })
           }),
-          _.map(workspace => {
-            return h(WorkspaceCard, { listView, workspace, key: workspace.workspace.workspaceId })
-          }, data),
-          !isDataLoaded && spinnerOverlay
-        ]),
+          listView ?
+            div({ style: { flex: 1, minWidth: 0, margin: '0.75rem 1rem 0rem 0.75rem' } },
+              renderedWorkspaces
+            ) : renderedWorkspaces
+        ]), !isDataLoaded && spinnerOverlay,
```

Comment:
nit: For readability it's nice to stay consistent with line breaks. The spinner should probably be on its own line since it's a peer of the card container.